hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/insights.tsx

```diff
@@ -95,7 +95,7 @@ export class Insights extends React.Component<Record<string, object>, IInsightsS
     // If there is only one volume, bucket selection dropdown should not be disabled.
     const isBucketSelectionDisabled = !selectedVolumes ||
         (selectedVolumes &&
-            (selectedVolumes.length > 2 &&
+            (selectedVolumes.length >= 2 &&
```

Comment:
@smitajoshi12 thanks for the patch, however this could be more readable for any developer to easily understand if you could use the condition >1

Reply:
@devmadhuu 
Hi Devesh Done changes in recent commit as per suggestion.